README.md

```diff
@@ -81,11 +81,24 @@ Calls `recorder.reset()` internally.
 
 Resets nock state & state of recorder.
 
-#### `recorder.mode()`
+#### `recorder.getMode()`
 
 Get the recorder mode.
 
-Returns the mode uses for the current run if we're in a recording. Returns the current default mode otherwise.
+Returns the mode uses for the current run if we're in a recording.
+Throws an exception if the recorder is not started.
```

Comment:
This seemed more appropriate than returning `null`, since the most common use case for `getMode` would be something like this:
```
if (recorder.getMode() === 'live') {
  ...
}
```
In the case above, `null` would not match, but the recorder could then subsequently be started in live mode...